src/styles/components/_dropdown.scss

```diff
@@ -51,7 +51,7 @@ $dso-dropdown-hor-padding: 20px;
         font-weight: 400;
         line-height: 1.5;
         padding: 3px $dso-dropdown-hor-padding;
-        white-space: nowrap;
+        white-space: nowrap; /* stylelint-disable-line declaration-property-value-disallowed-list */
```

Comment:
willen we behalve de lint disabling ook nog een inhoudelijke reden weten?

Reply:
Lijkt mij wel, maar ik ga voor de 3 uitzonderingen niet terug in de tijd filosoferen waarom destijds `white-space: nowrap` is gebruikt. Wellicht dat #772 daar iets over kan zeggen?

Reply:
In #772 is inderdaad onderzocht waarom de uitzonderingen toegestaan en zelfs noodzakelijk zijn. @daverhelst kun jij dat bij die 3 uitzonderingen kort en bondig noteren?

Reply:
Deze nowrap vindt zijn oorsprong in bootstrap, hier is nog wel de vraag te stellen of deze echt noodzakelijk is.